### PR TITLE
[TRACING-3556] Document disable_keep_alives

### PR DESCRIPTION
**Pull request by iblancasa**
Version(s): 4.15, 4.14, 4.13, 4.12, 4.11

Issue: [TRACING-3556](https://issues.redhat.com/browse/TRACING-3556)

Link to docs preview: https://70668--ocpdocs-pr.netlify.app/openshift-enterprise/latest/otel/otel-configuring

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by max-cx — APPROVED** [review]
LGTM

**Review by IshwarKanse — APPROVED** [review]

**Comment by iblancasa** [comment]
Is there anything pending?

File: modules/otel-config-collector.adoc [hunk 1]
```diff
@@ -760,17 +760,19 @@ The OTLP HTTP exporter exports traces and metrics using the OpenTelemetry protoc
...
         traces:
           exporters: [otlphttp]
         metrics:
-          expoters: [otlphttp]
+          exporters: [otlphttp]
 ----
 <1> The OTLP HTTP endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
 <2> The client side TLS configuration. Defines paths to TLS certificates.
 <3> Headers are sent in every HTTP request.
+<4> If true, disables HTTP keep-alives. It will only use the connection to the server for a single HTTP request.
```
Comment (dfitzmau) [inline modules/otel-config-collector.adoc:775]:
```suggestion
<4> If true, disables HTTP keep-alive mode. The parameter only uses the connection to the server for a single HTTP request.
```

* On scanning the customer documentation, "keep-alive" is usually followed by another noun that better describes the component. I also suggest removing "it" as it's a vague referrent. I assume we are discussing the parameter "uses the connection..."?

**Comment by dfitzmau** [comment]
Hi @iblancasa . Thanks for raising the PR. I added one comment inline. I do not see any QE ack. Would you be OK to request an ack from QE?

**Comment by iblancasa** [comment]
> Hi @iblancasa . Thanks for raising the PR. I added one comment inline. I do not see any QE ack. Would you be OK to request an ack from QE?

There is one
> [IshwarKanse](https://github.com/IshwarKanse) approved these changes [4 days ago](https://github.com/openshift/openshift-docs/pull/70668#pullrequestreview-1842744537)

**Comment by dfitzmau** [comment]
> > Hi @iblancasa . Thanks for raising the PR. I added one comment inline. I do not see any QE ack. Would you be OK to request an ack from QE?
> 
> There is one
> 
> > [IshwarKanse](https://github.com/IshwarKanse) approved these changes [4 days ago](https://github.com/openshift/openshift-docs/pull/70668#pullrequestreview-1842744537)

Thanks, @iblancasa . I missed that :dodo: . I'll approve when the build passes. Looks like the Prow build has stalled on a few PRs.

**Comment by dfitzmau** [comment]
/remove-label peer-review-in-progress

/label peer-review-done

LGTM

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**